FreeRTOS/Demo/CORTEX_A5_SAMA5D3x_Xplained_IAR/AtmelFiles/usb/device/core/USBDDriver.c

```diff
@@ -472,7 +472,7 @@ static void USBDDriver_Test(const USBDDriver *pDriver, uint8_t test)
 
         case USBFeatureRequest_TESTJ:
             /*Test mode Test_J: */
-            /*Upon command, a port�s transceiver must enter the high-speed J state and remain in that */
+            /*Upon command, a ports transceiver must enter the high-speed J state and remain in that */

```

Comment:
Why is a new special character being added here. 
Displayed as `Â`

Reply:
Removed special characters in comments.